forgerock-openbanking-cdr-aspsp-as/forgerock-openbanking-cdr-aspsp-as-gateway/forgerock-openbanking-cdr-aspsp-as-gateway-server/src/main/java/com/forgerock/openbanking/aspsp/as/api/registration/dynamic/DynamicRegistrationApiController.java

```diff
@@ -283,7 +285,8 @@ public CDRScope() {
                             .getStringClaim(RegistrationTppRequestClaims.SOFTWARE_STATEMENT);
 
                     //Convert in json for convenience
-                    String registrationRequestJson = registrationRequestJws.getJWTClaimsSet().toJSONObject().toJSONString();
+                    String registrationRequestJson =
+                            JSONObjectUtils.toJSONString(registrationRequestJws.getJWTClaimsSet().toJSONObject());
```

Comment:
Yuk - what's your line width set to?! 😆

Reply:
120 - the forgerock standard I'm afraid ;-)

Reply:
https://wikis.forgerock.org/confluence/display/devcom/Java